app/jobs/make_phone_calls_job.rb

```diff
@@ -0,0 +1,28 @@
+class MakePhoneCallsJob < ApplicationJob
+  queue_as :phone_calls
+
+  require 'net/http'
+  require 'open-uri'
+
+  def perform(args)
+    # Find all the phone stores for which there an activecalendar for the current day of the week
+    # For each store, create a job to make the request
```

Comment:
This comment does not seem to reflect the actual code, since this is only handling a single store. Maybe remove the comment?

Reply:
Ups, this was supposed to be in the Service.